desktop/src-tauri/src/custom_protocol.rs

```diff
@@ -63,38 +198,28 @@ impl CustomProtocol {
             tauri::async_runtime::block_on(async {
                 info!("App opened with URL: {:?}", url_scheme.to_string());
 
-                let msg = CustomProtocol::parse(&url_scheme.to_string());
+                let request = UrlParser::parse(&url_scheme.to_string());
                 let app_state = app_handle.state::<AppState>();
+                if let Err(err) = request {
+                    send_ui_message(
+                        app_state,
+                    UiMessage::CommandFailed(err),
+                        "Failed to broadcast custom protocol message").await;
+                    return;
```

Comment:
needs ` #[cfg(not(target_os = "windows"))]` as well, similar to the other `handle_error` functions, otherwise a popup will show up about every second. Will need to look into that at some point